packages/cli/src/commands/project/command.ts

```diff
@@ -0,0 +1,67 @@
+import { Command } from '../help';
+import { packageName } from '../../util/pkg-name';
+
+export const projectCommand: Command = {
+  name: 'project',
+  description: 'Manage your Vercel Projects.',
+  arguments: [
+    {
+      name: 'command',
+      required: true,
+    },
+  ],
+  subcommands: [
+    {
+      name: 'ls',
+      description: 'Show all projects in the selected team/user',
+      arguments: [],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'add',
+      description: 'Add a new project',
+      arguments: [
+        {
+          name: 'name',
+          required: true,
+        },
+      ],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'rm',
+      description: 'Remove a project',
+      arguments: [
+        {
+          name: 'name',
+          required: true,
+        },
+      ],
+      options: [],
+      examples: [],
+    },
+  ],
+  options: [
+    {
+      name: 'next',
+      description: 'Show next page of results',
+      argument: 'MS',
+      shorthand: 'n',
```

Comment:
**suggestion (blocking):** ‏use capital

```suggestion
      shorthand: 'N',
```

@trek The shorthand for this option is capital.

https://github.com/vercel/vercel/blob/e43191b1866da70a3dab3815a3f2176942240ef3/packages/cli/src/commands/project/index.ts#L25

Reply:
https://github.com/vercel/vercel/pull/10460